diff --git a/helium/_impl/__init__.py b/helium/_impl/__init__.py
index cc7b86a..7307806 100644
--- a/helium/_impl/__init__.py
+++ b/helium/_impl/__init__.py
@@ -86,9 +86,7 @@ class APIImpl:
 			'options': firefox_options
 		}
 		if profile:
-			# This is Deprecated in the driver so only do it (and trigger the
-			# warnings) if the user requests it
-			kwargs['firefox_profile'] = profile
+			firefox_options.profile = profile
 		service_log_path = 'nul' if is_windows() else '/dev/null'
 		service = ServiceFirefox(log_path=service_log_path)
 		result = Firefox(service=service, **kwargs)
